skimage/io/_plugins/tifffile_plugin.py

```diff
@@ -1,6 +1,51 @@
+from tifffile import imread as tifffile_imread
+from tifffile import imwrite as tifffile_imwrite
+
 __all__ = ['imread', 'imsave']
 
-from tifffile import imwrite as imsave, imread as tifffile_imread
+
+def imsave(fname, arr, **kwargs):
+    """Load a tiff image to file.
+
+    Parameters
+    ----------
+    fname : str or file
+        File name or file-like-object.
+    arr : ndarray
+        The array to write
+    kwargs : keyword pairs, optional
+        Additional keyword arguments to pass through (see ``tifffile``'s
+        ``imwrite`` function).
+
+    Notes
+    -----
+    Provided by the tifffile library [1]_, and supports many
+    advanced image types including multi-page and floating point.
+
+    This implementation will set `photomotric='RGB'` when writing if the first
+    or last axis of arr has shape 3 or 4. To override this, explicitly
+    specify the photometric kwarg.
```

Comment:
```suggestion
    pass the photometric kwarg.
```
either 'specify' or 'explicitly pass' (right?)